src/components/PageHeader.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { Typography } from '@material-ui/core';
+
+const StyledHeader = styled(Typography)`
+  width: 90vw;
+  border-bottom: 2px solid;
+  margin-bottom: 1rem;
+`;
+
+export const PageHeader: FC = ({ children }) => <StyledHeader variant="h3">{children}</StyledHeader>;
```

Comment:
det ble kanskje ikke direkte _fantastisk_
![bilde](https://user-images.githubusercontent.com/6313468/92727956-3c31ba00-f370-11ea-8afb-8210d9152afe.png)

Reply:
kan du legge på mer padding over "Ny publikasjon"? Og endre font?

Reply:
har begynt på egen sak for font nå, så kan ta det der. tenker vi må h1 på de der, og size ned slik at størrelsen passer (dette er h3 nå), så setter vi fonter direkte i theme og ikke direkte hvor de brukes
men kan legge til `margin-top: 1rem` for litt mer luft